storyblok/storyblok-design-system
# chore: update to vue 2.7

## Pull request type

Jira Link: -




## How to test this PR


This updates the DS to vue version 2.7

`gco chore/update-vue-2-7` test by checking out this branch and see if everything works fine in the application itself


## What is the new behavior?



- 
- 
- 

## Other information
